### ISSUE TITLE
[Feature] Weather forecast 

### ISSUE THREAD

**Issue by ToinneGuust** [pawelmalak/flame#301]
Would be great if we can have weather forecast for the next 3 days. (Now => Tomorrow => The day after tomorrow).

Also chance of rain added to "Additional weather data" would be interesting. The data is available from weatherapi.com

**Comment by ryankrage77** [comment]
Also noticed the weather doesn't update except when saving.

**Comment by altered-beast** [comment]
can confirm the weather doesnt update unless i save

**Comment by johncadengo** [comment]
Yes, also experiencing a static weather status. Is there something wrongly configured or is it a bug?

**Comment by pawelmalak** [comment]
> Also noticed the weather doesn't update except when saving.

Was it working for you in previous versions?
